src/Microsoft.AspNet.Razor/CodeGenerators/CSharpTagHelperCodeRenderer.cs

```diff
@@ -389,67 +410,64 @@ protected virtual string GenerateUniqueId()
             }
         }
 
-        private void RenderUnboundHTMLAttributes(IEnumerable<KeyValuePair<string, Chunk>> unboundHTMLAttributes)
+        private void RenderUnboundAttribute(KeyValuePair<string, Chunk> chunkAttribute)
         {
-            // Build out the unbound HTML attributes for the tag builder
-            foreach (var htmlAttribute in unboundHTMLAttributes)
+            string textValue = null;
+            var isPlainTextValue = false;
+            var attributeValue = chunkAttribute.Value;
```

Comment:
`attributeValueChunk` was a better name. Ideally stop passing around the poorly-named `chunkAttribute` and determine `attributeName` and `attributeValueChunk` in `RenderAttributes()` then pass those around instead.